docs/data/charts/getting-started/getting-started.md

```diff
@@ -32,44 +28,49 @@ pnpm add @mui/x-charts@next
 </codeblock>
 
 :::info
-With Next.js you might face the following error:
+The `next` tag is used to download the latest, **pre-release**, v7 version.
+:::
```

Comment:
The place was to be coherent with other packages. But this one is fine too

Reply:
I'll make sure the others are consistent as well! 
Took advantage of this callout, though, to clarify even further about the `next` tag 😬